packages/react/src/components/form/Radio/Radio.stories.tsx

```diff
@@ -39,12 +39,23 @@ const Template = (args: RadioProps) => {
   };
 
   return (
-    <Radio {...args} onChange={handleChange}>
-      <Radio.Element label="Blue squad" value="blue" variant="default" />
+    <Radio
```

Comment:
@ofreyssinet-ledger  What about separating this story into two sections and adding a title for each section?
This story is the first one we see when you open the documentation tab.

![image](https://user-images.githubusercontent.com/33158502/140786446-08a38332-a5fb-462a-94b7-a46748591583.png)

Reply:
will do that after the sprint planning 👍